Dockerfile

```diff
@@ -42,10 +42,22 @@ FROM eclipse-temurin:17-jdk-focal as main
 
 RUN apt-get update && apt-get install -y libjemalloc-dev
 
+# Install Python and pip and clean up apt cache
+RUN apt-get update && apt-get install -y python3 python3-pip && rm -rf /var/lib/apt/lists/*
```

Comment:
Why is it important to delete `/var/lib/apt/lists/*`? Is it just to reclaim space? Also can we do this in a nicer way using `apt` commands (instead of `rm -rf`)?